app/controllers/admin/publisher_notes_controller.rb

```diff
@@ -0,0 +1,75 @@
+module Admin
+  class PublisherNotesController < AdminController
+    before_action :authorize, except: :create
```

Comment:
Do we need this line?

Reply:
This prevents users who did create the comment from updating and deleting it via a `PATCH` or `DELETE` request